Library/Homebrew/requirements/ruby_requirement.rb

```diff
@@ -12,7 +12,20 @@ def initialize(tags)
     which_all("ruby").detect do |ruby|
       version = /\d\.\d/.match Utils.popen_read(ruby, "--version")
       next unless version
-      Version.create(version.to_s) >= Version.create(@version)
+      version = version.to_s
+      if version.to_f > 1.9
```

Comment:
You could add the portable Ruby to the `PATH`, either at the user's command shell prompt, or see for example
https://github.com/Linuxbrew/brew/blob/master/Library/Homebrew/utils.rb#L196